ios/engine/KMEI/KeymanEngine/Classes/KeyboardRepository/APILexicalModelRepository.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  APILexicalModelRepository.swift
+//  KeymanEngine
+//
+//  Created by Randy Boring on 3/19/19.
+//  Copyright © 2019 SIL International. All rights reserved.
+//
+
+import Foundation
+import UIKit // for UIDevice
+
+public enum APILexicalModelFetchError: Error {
+    case networkError(Error)
+    case noData
+    case parsingError(Error)
+}
+
+public class APILexicalModelRepository: LexicalModelRepository {
+    private let languagesAPIURL = URLComponents(string: "https://api.keyman.com/cloud/4.0/languages")!
```

Comment:
If this class is for querying the API endpoint for lexical models, this should be
```
private let lexicalModelAPIURL = URLComponents(string: "https://api.keyman.com/model")!
```

You'll also need to append `?q=bcp47:` and then the language ID (BCP 47 tag)

Reply:
I've taken the suggestion.